sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -435,6 +439,40 @@ def process_pipelineparam(s):
         }
         self.loops_pipeline[group_name]['spec']['params'].append(sep_param)
 
+      if hasattr(sub_group, 'start') and sub_group.start is not None:
+        # start, end, step params should be added as a parameter
+        # isinstance(sub_group.start, dsl.PipelineParam)
+        def process_parameter(parameter):
+          parameter_value = str(parameter)
+          if isinstance(parameter, dsl.PipelineParam):
+            if parameter.op_name:
+              parameter_value = '$(tasks.' + parameter.op_name + '.results.' + sanitize_k8s_name(parameter.name) + ')'
+            else:
+              parameter_value = '$(params.' + parameter.name + ')'
+          return parameter_value
+
+        def get_spec_param_keys(param_specs):
```

Comment:
This function doesn't seem to be used...

Reply:
I was thinking to use this function to get me all the parameter names to pass down to the nested loop. Since that feature needs some big changes to the controller, I removed it for now.